styx-scheduler-service/src/main/java/com/spotify/styx/docker/WorkflowValidator.java

```diff
@@ -37,19 +32,6 @@ private WorkflowValidator() {
   }
 
   public static boolean hasDockerConfiguration(Workflow workflow) {
-    return workflow.configuration().dockerArgs().isPresent();
-  }
-
-  public static boolean hasDockerConfiguration(Workflow workflow, Storage storage) {
-    final Optional<List<String>> dockerArgs = workflow.configuration().dockerArgs();
-    if (!dockerArgs.isPresent()) {
```

Comment:
I believe this was an intentional removal to allow scheduling docker containers with no docker args?

Reply:
yes